schemas/latest/dbt_yml_files-latest.json

```diff
@@ -343,13 +301,51 @@
         "additionalProperties": false
       }
     },
+    "saved_queries": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": ["name", "description", "label", "query_params"],
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "description": {
+            "type": "string"
+          },
+          "exports": {
+            "type": "array",
+            "items": {
+              "$ref": "#/$defs/export"
+            }
+          },
+          "label": {
+            "type": "string"
+          },
+          "query_params": {
+            "type": "object",
+            "properties": {
+              "metrics": {
+                "$ref": "#/$defs/array_of_strings"
+              },
+              "dimensions": {
+                "$ref": "#/$defs/array_of_strings"
+              },
+              "where": {
+                "$ref": "#/$defs/array_of_strings"
```

Comment:
I wonder if it's worth checking with a regex match for `{{ Dimension` for a lot of these, because it feels like a common mistake mode to just provide the name of the dim without the wrapper. ([related](https://dbt-labs.slack.com/archives/C03KHQRQUBX/p1715149318025209))